packages/framework/ai-collab/src/test/explicit-strategy/integration.spec.ts

```diff
@@ -0,0 +1,187 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-internal-modules
+import { createIdCompressor } from "@fluidframework/id-compressor/internal";
+// eslint-disable-next-line import/no-internal-modules
+import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils/internal";
+import {
+	SchemaFactory,
+	SharedTree,
+	TreeViewConfiguration,
+	// eslint-disable-next-line import/no-internal-modules
+} from "@fluidframework/tree/internal";
+
+import { generateTreeEdits } from "../../explicit-strategy/index.js";
+
+import { initializeOpenAIClient } from "./utils.js";
+
+const sf = new SchemaFactory("Planner");
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export class Session extends sf.object("Session", {
```

Comment:
Do the classes in this test file really need to be exported? If not, I think that'll let us remove the eslint disables.

Reply:
Unresolving to keep the unresolved comments in this PR as follow-ups, I still see them exported.